Doc/library/collections.abc.rst

```diff
@@ -122,61 +122,60 @@ ABC                            Inherits from          Abstract Methods        Mi
 ============================== ====================== ======================= ====================================================
 :class:`Container` [1]_                               ``__contains__``
 :class:`Hashable` [1]_                                ``__hash__``
-:class:`Iterable` [1]_ [2]_                           ``__iter__``
+:class:`Iterable` [1]_                                ``__iter__``
 :class:`Iterator` [1]_         :class:`Iterable`      ``__next__``            ``__iter__``
-:class:`Reversible` [1]_       :class:`Iterable`      ``__reversed__``
-:class:`Generator`  [1]_       :class:`Iterator`      ``send``, ``throw``     ``close``, ``__iter__``, ``__next__``
-:class:`Sized`  [1]_                                  ``__len__``
-:class:`Callable`  [1]_                               ``__call__``
-:class:`Collection`  [1]_      :class:`Sized`,        ``__contains__``,
+:class:`Reversible` [1]_       :class:`Iterable`      ``__reversed__``,
+                                                      ``__iter__``
+:class:`Generator` [1]_        :class:`Iterator`      ``send``,               Inherited mixin :class:`Iterator` method,
```

Comment:
I would like the Generator as is. The "inherited" references are used only when the would copy a lot of methods.  The current form for this entry is more readable, useful, and easy to understand.